source/camera/basic/CaptureTutorial.md

```diff
@@ -111,74 +108,79 @@ We may choose to configure settings manually. For more information about what ea
 
 We can configure settings for an individual frame directly to the camera ([go to source][settings-url]).
 ```python
-with camera.update_settings() as updater:
-    updater.settings.iris = 20
-    updater.settings.exposure_time = datetime.timedelta(microseconds=8333)
-    updater.setting.brightness = 1
-    updater.setting.gain = 1
-    updater.setting.bidirectional = False
-    updater.setting.filters.contrast.enabled = True
-    updater.setting.filters.contrast.threshold = 5
-    updater.setting.filters.gaussian.enabled = True
-    updater.setting.filters.gaussian.sigma = 1.5
-    updater.setting.filters.outlier.enabled = True
-    updater.setting.filters.outlier.threshold = 5
-    updater.setting.filters.reflection.enabled = True
-    updater.setting.filters.saturated.enabled = True
-    updater.setting.blue_balance = 1.081
-    updater.setting.red_balance = 1.709
+settings = zivid.Settings(
+    acquisitions=[
+        zivid.Settings.Acquisition(
+            aperture=5.66, exposure_time=datetime.timedelta(microseconds=8333),
+        ),
+    ],
+    processing=zivid.Settings.Processing(
+        filters=zivid.Settings.Processing.Filters(
+            outlier=zivid.Settings.Processing.Filters.Outlier(
+                removal=zivid.Settings.Processing.Filters.Outlier.Removal(
+                    enabled=True, threshold=5
+                )
+            )
+        )
+    ),
+)
```

Comment:
Consider replacing with this if you want something more compact:
```
settings = zivid.Settings()
settings.acquisitions.append(zivid.Settings.Acquisition())
settings.acquisitions[0].aperture = 5.66
settings.acquisitions[0].exposure_time = datetime.timedelta(microseconds=8333)
settings.processing.filters.outlier.removal.enabled = True
settings.processing.filters.outlier.removal.threshold = 5.0
```

Reply:
done